docs/diagrams/ImportExportObjectDiagram.puml

```diff
@@ -0,0 +1,36 @@
+
+@startuml
+!include style.puml
+
+object "__model:ModelManager__" as model
+object "__stagedMemeList:ObservableList<Meme>__" as stagedList
```

Comment:
isnt it called exportList in Weme?

Reply:
Yes but it becomes stagedList in ModelManager. Seems like it is not so straightforward